telegram/_bot.py

```diff
@@ -2037,16 +2051,40 @@ async def send_media_group(
                 :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
             api_kwargs (:obj:`dict`, optional): Arbitrary keyword arguments to be passed to the
                 Telegram API.
+            group_caption (:obj:`str`, optional): Text of the caption
+                that will be shown for the whole media group.
+                Defaults to :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
+            group_caption_parse_mode (:obj:`str` | :obj:`None`, optional):
+                Parse mode of the caption that will be shown for the whole media group.
+                Defaults to :attr:`~telegram.request.BaseRequest.DEFAULT_NONE`.
```

Comment:
```suggestion
                Defaults to :obj:`None`.
```
So far, we rarely point to the `Defaults` setup in the docs of the bot methods. Instead, we could leave a note in the docs of the `Defaults.parse_mode` parameter stating that it also applies here :)

Reply:
Fixed in e4b7971164c4c08aab6920401bae0bd74397dcf7, but not sure I *completely* understand the difference between places where it `DEFAULT_NONE` is there and where it isn't. Maybe the difference is that the new arguments don't reflect Telegram API.